runtime/builtin_types_test.go

```diff
@@ -23,6 +23,33 @@ import (
 	"testing"
 )
 
+func TestBuiltinDelAttr(t *testing.T) {
+	fun := wrapFuncForTest(func(f *Frame, args ...*Object) (*Object, *BaseException) {
```

Comment:
This can be `func(*Frame, o *Object, args ...*Object) (*Object, *BaseException)`. Then you don't have to mess around with args slices below.